src/pages/hegel/guides/general-division-of-the-logic.mdx

```diff
@@ -0,0 +1,101 @@
+
+> Now because the interest of the Kantian philosophy was directed to the so-called _transcendental_ nature of the categories, the treatment itself of such categories came up empty. What they are in themselves apart from their abstract relation to the "I," a relation which is the same for all, how they are determined and related to each other, this was not made a subject of consideration, and therefore knowledge of their nature was not in the least advanced by this philosophy (Hegel, 2010, 41/21.48).
+
+If the treatment of the categories comes up empty, then there is essentially no developmental understanding at play and therefore `the concept` is itself lacking. Kant only stumbles, as it were, over the concept following his transcendental inquiry&mdash;but it speaks to the _power_ of Kant's thought that he is, despite systematic limitations, able to discover and appreciate the underlying conceptual movement necessarily at work in pure intuitions and categories of the understanding. 
+
+Finally, Hegel deals with the Kantian thing-in-itself as part of the dialectical development of logic, particularly in the Doctrine of Essence where this determination receives its treatment in both as the thing in itself and as the world in and for itself. Here it is seen how this determination necessarily follows from the determination of thing, and subsequently from the world that appears, but that it also then becomes later sublated in the following `essential relation` between the world that appears and the world that is in and for itself. Now, this actually serves to vindicate Kant somewhat since the idea of the thing-in-itself, even in Hegel's own estimate, necessary follows from the logical development and therefore cannot be, as Hegel repeatedly writes, something external or alien to thinking.[^1]
+
+[^1]: One could come to Hegel's defense here and say that it is _how_ Kant treats the thing-in-itself which makes abstract and alien to thought, but given Hegel's frequent assaults on the very idea of a thing-in-itself, this defense seems weak.
+
+### The Objective Logic Replaces Former Metaphysics
+
+Hegel states that the objective logic takes the place of former metaphysics inasmuch as that was a metaphysics that understood reality purely in terms of ideas and principles. More specifically, Hegel's objective logic aims to supplant ontology in the classical sense of being the philosophical treatment of things (_ens_). There are, however, a number of other ideas and principles that the older metaphysics were very concerned with but that Hegel's ontology will jettison, such as the soul, the world, God, and others.[^2] In Hegel's estimate, these ideas are subjects of _figurative imagination_ rather than categories of thought and his aim is to conduct his examination of the categories of thought free from the intermingling of figurative or representational content. 
```

Comment:
Perhaps this is an unnecessary point, but Hegel does explicitly treat soul in the PS, as you know.

Reply:
Phenomenology of Spirit?

Reply:
Well, I would want to say that the philosophy of spirit is a continuation of the study of ontology from the Logic. I think we probably do need to flag this because it's probably a rather big issue. We might be able to side-step it by just deleting that sentence that includes specific examples of ideas and principles that Hegel jettisons.

Reply:
Yeah, though, this is what H writes in the text - that he drops certain traditional themes from metaphysics, and I think he is broadly right, even though, as you say, they all do come back later in the system. I already have a footnote where I challenge H's claim about the world, since that's a category explicitly examined in Essence. How about I expand that footnote with the stuff about God and soul, that these do come back in Hegel's system, but later, and I can also raise a warning as to how to understand ontology generally?

Reply:
Looks good to me.